var/spack/repos/builtin/packages/sherpa/package.py

```diff
@@ -0,0 +1,129 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Sherpa(AutotoolsPackage):
+    """Sherpa is a Monte Carlo event generator for the Simulation of
+    High-Energy Reactions of PArticles in lepton-lepton, lepton-photon,
+    photon-photon, lepton-hadron and hadron-hadron collisions."""
+
+    homepage = "https://sherpa-team.gitlab.io"
+    url      = "https://gitlab.com/sherpa-team/sherpa/-/archive/v2.2.11/sherpa-v2.2.11.tar.gz"
+    list_url = "https://gitlab.com/sherpa-team/sherpa/-/tags"
+    git      = "https://gitlab.com/sherpa-team/sherpa.git"
+
+    tags = ['hep', 'eic']
+
+    maintainers = ['wdconinc', 'vvolkl']
+
```

Comment:
```suggestion
    version('2.2.12', sha256='4ba78098e45aaac0bc303d1b5abdc15809f30b407abf9457d99b55e63384c83d')
```

Reply:
Thanks @wdconinc, the new version builds ok for me. I'll try to add some of the examples as tests in a future PR.